plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -887,6 +887,17 @@ public void testSystemSchemas()
         assertQueryReturnsEmptyResult("SHOW SCHEMAS IN mongodb LIKE 'local'");
     }
 
+    @Test
+    public void testInvalidCreateTableWithDuplicatedRowFields()
+    {
+        assertQueryFails(
+                "create table t3 (a row(a int, a int, c int, b int))",
+                "Row field 'a' specified more than once");
+        assertQueryFails(
+                "create table t3 (a row(a int, a int, c int, b int, b int))",
+                "Row fields 'a','b' specified more than once");
```

Comment:
Please create a table directly in MongoDB with `_schema` collection and try `INSERT` statement in Trino.

Reply:
1. I added tests for case sensitivity and nested rows
2. For inserting table directly into schema, what's the expected outcome? incorrect results are returned (as they are now), or we validate insert or select if table definition is not correct?

Reply:
I covered few more use cases, like row nested in array and map

Reply:
Regarding 2, insert/select should fail.